swagger/v1/api_spec.json

```diff
@@ -880,6 +880,58 @@
         }
       }
     },
+    "/api/v1/participants/npq/outcomes": {
+      "get": {
+        "summary": "List all participant declarations",
+        "operationId": "participant_outcomes",
+        "tags": [
+          "participants outcomes"
+        ],
+        "security": [
+          {
+            "bearerAuth": [
+
+            ]
+          }
+        ],
+        "parameters": [
+          {
+            "name": "page",
+            "in": "query",
+            "schema": {
+              "$ref": "#/components/schemas/Pagination"
+            },
+            "style": "deepObject",
+            "explode": true,
+            "required": false,
+            "example": "page[page]=1&page[per_page]=5",
+            "description": "Pagination options to navigate through the list of participant declarations."
```

Comment:
```suggestion
            "description": "Pagination options to navigate through the list of participant NPQ outcomes."
```

Reply:
Updated in https://github.com/DFE-Digital/early-careers-framework/pull/2782/commits/bc846d65c22fdb9b4699ab415473fa65ed963b21